osu.Game/Rulesets/UI/GameplayCursorContainer.cs

```diff
@@ -17,10 +19,30 @@ public partial class GameplayCursorContainer : CursorContainer
         /// This holds the true visibility value.
         /// </summary>
         public Visibility LastFrameState;
+        public bool Delayed;
+        public Vector2 TruePosition = Vector2.Zero;
+        public Vector2 TargetPosition = Vector2.Zero;
+		public float FollowSpeed;
+
+        protected override bool OnMouseMove(MouseMoveEvent e)
+        {
+            TargetPosition = e.MousePosition;
+            return true;
+        }
 
         protected override void Update()
         {
+            // FIXME: This is jank-tastic because I don't know how to modify osu!framework. But it works.
             base.Update();
+            if (Delayed)
+            {
+                TruePosition = Vector2.Lerp(TruePosition, TargetPosition, (float)Time.Elapsed / 1000 * FollowSpeed);
+            }
+            else
+            {
+                TruePosition = TargetPosition;
+            }
+            ActiveCursor.Position = TruePosition;
```

Comment:
What is the point of `TruePosition` here? Can't you just set `ActiveCursor.Position` directly?